### ISSUE TITLE
Nix installation failing with infinite recursion error.

### ISSUE THREAD

**Issue by lfearnley** [vlaci/openconnect-sso#68]
Nix setup is failing; the script generates an infinite recursion.

```
nix-env -i -f https://github.com/vlaci/openconnect-sso/archive/master.tar.gz
installing 'python3.9-openconnect-sso-0.7.3'
error: infinite recursion encountered

       at /nix/store/f6kxwf008fylybbw7p18m86m38fq9bbp-nixpkgs-21.11pre329221.c11d08f0239/nixpkgs/pkgs/top-level/python-aliases.nix:87:20:

           86|   selectors34 = throw "selectors34 has been removed: functionality provided by Python itself; archived by upstream."; # added 2021-06-10
           87|   setuptools_scm = setuptools-scm; # added 2021-06-03
             |                    ^
           88|   smart_open = smart-open; # added 2021-03-14
(use '--show-trace' to show detailed location information)
```

**Comment by jbedo** [comment]
I can reproduce this on the nixos-unstable channel. Works fine against the nixos-21.05 stable channel.

**Comment by vlaci** [comment]
Should be fixed on 0.8.0. Unfortunately I cannot test it. Feel free to reopen if the issue persist.

**Comment by kczulko** [comment]
@vlaci 
Thank you very much for this project. It rocks!

I also encountered the same error while upgrading my nixos 21.05->21.11

And I want to confirm that you're right, it's fixed in 0.8.0. 

Best regards,
Karol
